packages/macros/src/ember-addon-main.ts

```diff
@@ -70,6 +72,30 @@ export = {
       let appInstance = this._findHost();
       let source = appOrAddonInstance.root || appOrAddonInstance.project.root;
       babelPlugins.unshift(MacrosConfig.for(appInstance).babelPluginConfig(source));
+
+      let yarnLockPath = join(appInstance.project.root, 'yarn.lock');
+      let npmLockPath = join(appInstance.project.root, 'package-lock.json');
+      let packagePath = join(appInstance.project.root, 'package.json');
+      let lockFileBuffer;
```

Comment:
Mind adding `pnpm-lock.yaml` also? Some users are succesfully using `pnpm`, and it would be nice to support _basically_ all popular lockfile formats...

Reply:
Added!